src/server/utilities/cspHeader/index.js

```diff
@@ -63,7 +73,10 @@ const advertisingDirectives = {
     'https://tpc.googlesyndication.com',
     'https://gn-web-assets.api.bbc.com',
     'https://www.googletagservices.com',
+    'https://securepubads.g.doubleclick.net',
+    'https://bbc.gscontxt.net',
```

Comment:
Duplication of line 61?

Reply:
This is for testing: isAmp: true, isLive: true.